docs/scripting-ref.rst

```diff
@@ -917,6 +917,84 @@ If your script returns the result of ``splash:png()`` in a top-level
 ``"png"`` key (as we've done in a previous example) then Splash UI
 will display it as an image.
 
+.. _splash-jpeg:
+
+splash:jpeg
+-----------
+
+Return a `width x height` screenshot of a current page in JPEG format.
+
+**Signature:** ``jpeg = splash:jpeg{width=nil, height=nil, render_all=false, scale_method='raster', quality=75}``
+
+**Parameters:**
+
+* width - optional, width of a screenshot in pixels;
+* height - optional, height of a screenshot in pixels;
+* render_all - optional, if ``true`` render the whole webpage;
+* scale_method - optional, method to use when resizing the image, ``'raster'``
+  or ``'vector'``
+* quality - optional, quality of JPEG image, integer in range from ``0`` to ``100``
+
+**Returns:** JPEG screenshot data.
+
+**Async:** no.
+
+Without arguments ``splash:jpeg()`` will take a snapshot of the current viewport.
```

Comment:
I wonder if we actually need all of that documentation for JPEG endpoint. I mean, if anything changes we'll have one more place to fix it. Maybe we could refer to relevant portions of `splash:png` and just underline the differences?